server/src/main/java/com/epam/aidial/core/server/validation/DialFileFormat.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.aidial.core.server.validation;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.networknt.schema.ExecutionContext;
+import com.networknt.schema.Format;
+import com.networknt.schema.JsonType;
+import com.networknt.schema.TypeFactory;
+import com.networknt.schema.ValidationContext;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DialFileFormat implements Format {
+
+    private static final Pattern PATTERN = Pattern.compile("files/[a-zA-Z0-9]+/.*");
```

Comment:
Does it match with "some-text files/x/y.txt"?

Reply:
Should not. Fixed.